_posts/12-05-01-Building-your-Application.md

```diff
@@ -44,7 +44,7 @@ developers interested in Capistrano.
 [Chef] is more than a deployment framework, it is a very powerful Ruby based system integration framework that doesn't
 just deploy your app but can build your whole server environment or virtual boxes.
 
-[Deployer] is a deployment tool written in PHP, it's simple and functional. Deploy your code to all servers you want, it supports deploy via copy, or via VCS (like git), or via rsync. Run your tasks on all your servers, or use our recipes of common tasks for Symfony, Laravel, Zend Framework and Yii.
+[Deployer] is a deployment tool written in PHP, it's simple and functional. Run tasks in parallel, atomic deployment, keep servers in consistency. Recipes of common tasks for Symfony, Laravel, Zend Framework and Yii.
```

Comment:
I'm not entirely sure if "keeps servers in consistency"  is valid English. I am pretty bad at words though so hey.